### PR TITLE
CI tooling [#3]

### PR DESCRIPTION
**Pull request by marnen**
Pull request for #3. This seems to work well locally, but we'll have to have Travis run this in the course of the PR, I guess.

### CODE REVIEW

**Comment by marnen** [comment]
Oops! Apparently I forgot to have Travis install APL on the CI server. Fixing that...

**Comment by marnen** [comment]
Awesome. This seems to work.

**Comment by marnen** [comment]
Unfortunately I can't get Travis to show the proper ANSI colors for the APL output; this is how it looks in my dev environment: 
image

@kytrinyx If you know how to do that, I'd appreciate some advise.

File: run-all-tests [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+STATUS=0
+APL=/usr/bin/apl
+APL_OPTS="--silent --Color"
```
Comment (marnen) [inline run-all-tests]: I should have used single quotes.

File: run-all-tests [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (marnen) [inline run-all-tests]: I wonder if this script should go in `./bin`.

Comment (kytrinyx) [reply run-all-tests]: That seems like a clean way to do it, but I don't have any strong opinions either way.

**Comment by kytrinyx** [comment]
> If you know how to do that

I don't, unfortunately. That said... I have friends who work at Travis. I'll ask.

**Comment by marnen** [comment]
It might just be an issue with Travis, but I thought I remembered that RSpec output showed up with the proper colors in Travis...

**Comment by marnen** [comment]
I just realized that some of this color stuff might be in an environment config file. It's not a very high priority compared to other things I want to change here, but I'll have a look. Reference: https://www.gnu.org/software/apl/apl.html#Section-3_002e6

**Comment by marnen** [comment]
I think this is ready to merge. @kytrinyx Any further thoughts?

**Pull request merged**